packages/components/README.md

```diff
@@ -5,50 +5,223 @@
+
+```js
+import '@plone/components/basic.css';
+```
+
+or selectively:
+
+```js
+import '@plone/components/src/styles/basic/TextField.css';
+```
+
+## Theming
+
+You can use the basic styles as a baseline while building the theme of your site.
+You can take advantage of them, as they are very thin and basic (almost vanilla CSS for the components).
+Using them as a baseline will allow you to quickly build your theme around them.
+`@plone/components` basic styles provide a simple, yet powerful, set of tokenized custom CSS properties that will help you customize your own styles on the top of the basic styling.
+You can override them in your classes while maintaining them for others.
+
+### Quanta
+
+This package also features the Quanta components.
+The Quanta theme is an example of it.
+These components use the basic styling as a baseline, not only in styling, but also in the component side, reusing the CSS and custom CSS properties in it.
+
+Quanta is built upon the basic styles in an additive way.
+The use of the Quanta CSS implies using it upon basic styling.
+You could take Quanta as example to build your own layer of styles over basic styling for your theme.
+
+To use a theme built upon the basic styling, you need to import both the basic and the theme CSS, in this order:
+
+```js
+import '@plone/components/basic.css';
+import '@plone/components/quanta.css';
+```
+
+You have the option of doing it selectively per component, too:
+
+```js
+import '@plone/components/src/styles/basic/TextField.css';
+import '@plone/components/src/styles/quanta/TextField.css';
+```
+
+Take a look at the implementation of the Quanta components, using the basic ones as baseline in the `quanta` folders.
+
+Alternatively, as RAC allows, you can also drop your own basic set of styles.
+You can take the basic styles as reference.
+You can even bring your own CSS framework and make basic get the styling in there using the utilities of your choice.
+It's even possible to use TailwindCSS for styling the components in this package, using the RAC styling approach.
 
 ## Components list
 
-### Fundamentals
+### Basic
 
+- Button
+- Checkbox
+- CheckboxField
+- CheckboxGroup
+- Combobox
 - Container
-- Input
+- Dialog
+- Form
+- GridList
 - Icon
-
-### Structural
+- Link
+- Listbox
+- Menu
+- Modal
+- NumberField
+- Popover
+- RadioGroup
+- Select
+- Switch
+- Tabs
+- TagGroup
+- TextField
+- TextAreaField
+- ToggleButton
+- Toolbar
+- BlockToolbar
+- Tooltip
+
+### Viewlets
 
 - Breadcrumbs
 
+### Views
+
+- DefaultBlockView
+- RenderBlocks
+
+### Quanta
+
+- TextField
+- TextAreaField
+
+## Quanta icons
+
+This package provides an implementation of the Quanta Icon set in React components.
+They can be used directly in your components as:
+
+```tsx
+import { ChevronupIcon } from '../Icons/ChevronupIcon';
+
+const MyComponent = (props) => (
+  <Button aria-label="Unfold/Collapse">
+    {props.isOpen ? <ChevronupIcon /> : <ChevrondownIcon />}
```

Comment:
@sneridagh I know this is just demo example code but there is no import of ChevrondownIcon,
or where Button comes from. We can keep examples to minimum but I still think they should be
as close to working as possible which to me means that this example is missing the Button and
the ChevrondownIcon import